### PR TITLE
fixes #7224 feat(nimbus): show targeting description

### LINKED ISSUE #7224: Expose description field when selecting targeting config
When the user selects the targeting config, they don't see the description of that config, so in this task add a description in the dropdown select value for targeting config.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2391)

### PR DESCRIPTION
**Pull request by yashika-khurana**
Because

* We have option to set the `Advanced Targeting` using the drop down on `Audience page`, but missing the description for that

This commit

* Adds the `description` in the drop-down along with the label

Before changes screenshot
image

After changes screenshot
image

closes #7224

### CODE REVIEW

**Comment by jaredlockhart** [comment]
@yashika-khurana This looks like it has many changes in one PR, should this be open for review?

**Comment by yashika-khurana** [comment]
> @yashika-khurana This looks like it has many changes in one PR, should this be open for review?

@jaredlockhart its just taking the commits from the existing branch, I created the sub-branch while I was waiting for the review, do you want to be separate out?

**Comment by jaredlockhart** [comment]
@yashika-khurana Oh yeah that's what I figured, I forgot to mention, if you create a PR branched off of another PR, just add the `Blocked` label and add a comment saying `blocked on landing <link to other pr>` so it's clear what the order is 👍

File: app/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx [hunk 1]
```diff
@@ -116,6 +116,18 @@ const TableOverview = ({
                 )}
               </td>
             </tr>
+            <tr>
+              <th>Advanced Targeting</th>
+              <td data-testid="experiment-targeting-config">
+                {experiment.targetingConfig?.length ? (
+                  experiment.targetingConfig.map((t) => (
+                    <p key={t?.label}>{t?.label + "- " + t?.description}</p>
```
Comment (jaredlockhart) [inline app/experimenter/nimbus-ui/src/components/Summary/TableOverview/index.tsx:124]: Format string and spacing here pls 🙏

**Review by jaredlockhart — CHANGES_REQUESTED** [review]

**Comment by yashika-khurana** [comment]
> @yashika-khurana Oh yeah that's what I figured, I forgot to mention, if you create a PR branched off of another PR, just add the `Blocked` label and add a comment saying `blocked on landing <link to other pr>` so it's clear what the order is 👍

got it, will take care 👍

**Comment by jaredlockhart** [comment]
I'm getting confused about which pr covers which change 😅

**Comment by yashika-khurana** [comment]
> I'm getting confused about which pr covers which change 😅

> I'm getting confused about which pr covers which change 😅

side-effect of sub-branch into sub-branch, it should be good now 😅, sorry for the confusions

**Review by jaredlockhart — APPROVED** [review]
LGTM 👍 🎉 🎉 🎉

**Pull request merged**